From: =?utf-8?q?Rapha=C3=ABl_Hertzog?= <hertzog@debian.org>
Date: Wed, 24 Jun 2015 17:39:55 +0200
Subject: Do not pass the version to apt-ftprelease when we don't have one

---
 scripts/build/installer_debian-installer | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/scripts/build/installer_debian-installer b/scripts/build/installer_debian-installer
index 1f137eb..f8457b1 100755
--- a/scripts/build/installer_debian-installer
+++ b/scripts/build/installer_debian-installer
@@ -731,6 +731,9 @@ then
 	mv binary chroot/root
 
 	eval _VERSION="$`echo RELEASE_${LB_PARENT_DISTRIBUTION}`"
+	if [ -n "${_VERSION}" ]; then
+		_LB_APT_VERSION_OPT='-o APT::FTPArchive::Release::Version="'"${_VERSION}"'"'
+	fi
 
 	case "${LB_PARENT_DISTRIBUTION}" in
 		sid)
@@ -747,7 +750,7 @@ cd /root/binary && apt-ftparchive \
 	-o APT::FTPArchive::Release::Origin="Debian" \
 	-o APT::FTPArchive::Release::Label="Debian" \
 	-o APT::FTPArchive::Release::Suite="${_SUITE}" \
-	-o APT::FTPArchive::Release::Version="${_VERSION}" \
+	${_LB_APT_VERSION_OPT} \
 	-o APT::FTPArchive::Release::Codename="${LB_PARENT_DISTRIBUTION}" \
 	-o APT::FTPArchive::Release::Date="$(date -R)" \
 	-o APT::FTPArchive::Release::Architectures="${LB_ARCHITECTURES}" \
