From: =?utf-8?q?Rapha=C3=ABl_Hertzog?= <hertzog@debian.org>
Date: Thu, 18 Jun 2015 10:38:42 +0200
Subject: Use linux-image-586 instead of linux-image-486 for kali 2

---
 scripts/build/installer_debian-installer | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/scripts/build/installer_debian-installer b/scripts/build/installer_debian-installer
index 5f6b3ca..1f137eb 100755
--- a/scripts/build/installer_debian-installer
+++ b/scripts/build/installer_debian-installer
@@ -369,7 +369,14 @@ then
 					;;
 
 				*)
-					DI_PACKAGES="${DI_REQ_PACKAGES} linux-image-486 linux-image-686-pae"
+					case "${LB_DISTRIBUTION}" in
+						kali-dev|kali-rolling|kali-next|sana)
+							DI_PACKAGES="${DI_REQ_PACKAGES} linux-image-586 linux-image-686-pae"
+							;;
+						*)
+							DI_PACKAGES="${DI_REQ_PACKAGES} linux-image-486 linux-image-686-pae"
+							;;
+					esac
 					;;
 			esac
 			;;
